### ISSUE TITLE
Accessibility > Links missing labels/name

### ISSUE THREAD

**Issue by tiarebalbi** [tiarebalbi/tiarebalbi-website#292]
Failing Elements
<img width="942" alt="Screen Shot 2021-03-09 at 6 21 53 AM" src="https://user-images.githubusercontent.com/103419/110427494-bf259a00-809f-11eb-9bcd-71dc43c81413.png">

--
a > <a href="/">
a > <a target="_blank" rel="noreferrer" href="https://github.com/tiarebalbi">
a > <a target="_blank" rel="noreferrer" href="https://twitter.com/tiarebalbi">
a > <a target="_blank" rel="noreferrer" href="https://ie.linkedin.com/in/tiarebalbi">
